include/zephyr/drivers/can.h

```diff
@@ -854,8 +862,17 @@ static inline uint32_t z_impl_can_get_bitrate_min(const struct device *dev)
  *
  * @deprecated Use @a can_get_bitrate_min() instead.
  *
+ * @note The minimum bitrate represents limitations of the CAN controller/transceiver
```

Comment:
It's laudable to update the docs for the deprecated function -- it really is! -- but in this instance I wonder if it's not counter productive. Keeping the docs of the deprecated method as lean as possible helps make the `@deprecated` mention less buried in surrounding text.
I think it's acceptable to let the docs of a deprecated method go stale since, after all, any new user is encouraged to look for the replacement. And to some extent I think it's ok that any existing user in need to look back at the docs to understand/clarify the behavior would basically see the docs as they were when it was decided to deprecate the method.

Reply:
Thank you for reviewing. I opted to keep the documentation in sync between the deprecated functions and their replacement to convey that there are no functional changes between the two.